qutebrowser/misc/tabsfinder.py

```diff
@@ -0,0 +1,43 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
```

Comment:
This should also have the license comment other files have (with 2019 as the year).